observability/arobit/deploy/values.yaml

```diff
@@ -22,171 +22,9 @@ forwarder:
     ##
     image:
       registry: mcr.microsoft.com
-      repository: /oss/fluent/fluent-bit
-      digest: sha256:667535f49ba225d96395ec8df3dcf9cf5f946facdb69afe1d920ebba3e7a4265
+      repository: /oss/v2/fluent/fluent-bit
+      digest: sha256:a4345e2041f03ef869dafdf7e21f985a35bdb084363d1b6330c7e37361f8d0ef
       pullPolicy: IfNotPresent
-    ## The configMap that contains the configuration files for fluent-bit
-    ##
-    configMap:
```

Comment:
moved to fowarder-configmap.yaml, cause that enables usage of templating in the config